laravel/resources/views/layouts/nav-main.blade.php

```diff
@@ -0,0 +1,88 @@
+    <div class="container">
+        <a class="navbar-brand" href="{{Route('dashboard')}}">Dynamic Banner</a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse justify-content-end text-bg-dark">
+            <ul class="navbar-nav my-auto">
+                <li class="nav-item">
+                    <a class="nav-link {{ (request()->is('login') ? 'active' : '') }}" href="{{Route('login')}}">Login</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="{{ route('password.request') }}">Forgot Password</a>
+                </li>
+            </ul>
+        </div>
+    </div>
+</nav>
+@endguest
+@auth()
+<nav class="navbar navbar-dark bg-dark navbar-expand-lg">
+    <div class="container">
+        <a class="navbar-brand" href="{{Route('dashboard')}}">Dynamic Banner</a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse">
+            <ul class="navbar-nav">
+                <li class="nav-item">
+                    <a class="nav-link {{ (str_starts_with(Route::currentRouteName(), 'dashboard')) ? 'active' : '' }}" aria-current="page" href="{{Route('dashboard')}}">Dashboard</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link {{ (str_starts_with(Route::currentRouteName(), 'instance')) ? 'active' : '' }}" aria-current="page" href="{{ Route('instances') }}">Instances</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link {{ (str_starts_with(Route::currentRouteName(), 'template')) ? 'active' : '' }}" aria-current="page" href="{{ Route('templates') }}">Templates</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link {{ (str_starts_with(Route::currentRouteName(), 'banner')) ? 'active' : '' }}" aria-current="page" href="{{ Route('banners') }}">Banners</a>
+                </li>
+            </ul>
+        </div>
+        <div class="collapse navbar-collapse justify-content-end text-bg-dark">
+            <ul class="navbar-nav my-auto">
+                <li class="nav-item dropdown">
+                    <a class="nav-link dropdown-toggle
+                    {{ (str_starts_with(Route::currentRouteName(), 'administration.')) ? 'active' : '' }}"
+                       href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+                        Administration
```

Comment:
[CONCERN]

What if the user does not have any administrative permissions? Then this drop-down menu will be empty.

I would add here a `@canAny()` or similar.